ch_tools/monrun_checks_keeper/main.py

```diff
@@ -71,9 +66,7 @@ def wrapper(ctx, *a, **kw):
                 status.set_code(1)
 
             log_message = f"Completed with {status.code};{status.message}"
-            logging.log(
-                logging.DEBUG if status.code == 0 else logging.ERROR, log_message
-            )
+            logging.log_status(status.code if status.code <= 1 else 2, log_message)
```

Comment:
Suggest to replace with
```
logging.log_status(status.code, log_message)
```

The old logic looks wrong / doesn't handle warnings correctly.